src/libraries/System.IO.Compression/src/System/IO/Compression/DeflateManaged/InflaterManaged.cs

```diff
@@ -18,15 +18,15 @@ internal sealed class InflaterManaged
 
         // The base length for length code 257 - 285.
         // The formula to get the real length for a length code is lengthBase[code - 257] + (value stored in extraBits)
-        private static readonly int[] s_lengthBase =
+        private static ReadOnlySpan<int> LengthBase => new int[]
         {
             3, 4, 5, 6, 7, 8, 9, 10, 11, 13, 15, 17, 19, 23, 27, 31, 35, 43, 51,
             59, 67, 83, 99, 115, 131, 163, 195, 227, 3
         };
 
         // The base distance for distance code 0 - 31
         // The real distance for a distance code is  distanceBasePosition[code] + (value stored in extraBits)
-        private static readonly int[] s_distanceBasePosition =
+        private static ReadOnlySpan<int> DistanceBasePosition => new int[]
```

Comment:
Can these be `byte` and `ushort` respectively?

And in general would it be worth it to use the smallest possible integer type for size savings?

Reply:
They can be.

There is a size benefit to using a smaller type.  In general I haven't audited to make such changes, as that's not a new benefit; the data for these as arrays and as spans is stored almost exactly the same way.